setup.py

```diff
@@ -49,8 +49,8 @@
         ],
         "blendsearch": ["flaml[blendsearch]"],
         "mathchat": ["sympy", "pydantic==1.10.9", "wolframalpha"],
-        "retrievechat": ["chromadb", "sentence_transformers", "pypdf", "ipython"],
-        "teachable": ["chromadb"],
+        "retrievechat": ["lancedb", "sentence_transformers", "pypdf", "ipython"],
+        "teachable": ["lancedb"],
```

Comment:
Will this break teachable agent? @rickyloynd-microsoft

Reply:
Yes, teachable should still use chromadb. TeachableAgent has no dependencies on the retrieval code.

Reply:
reverted